diff --git a/_includes/head.liquid b/_includes/head.liquid
index 37b808e9d0b5edc0b5bb7ed3e1ade24bc3443f4d..4e5c6781d55fe9b44fd683b9d29fafd59432cb13 100644
--- a/_includes/head.liquid
+++ b/_includes/head.liquid
@@ -5,7 +5,7 @@
   <meta name="keywords" content="JavaScript, Linter, Linting, Pluggable, Configurable, Code Quality">
   <meta name="description" content="{{ site.description }}">
   <meta name="theme-color" content="#463fd4">
-  <meta property="og:locale" content="en_US">
+  <!-- <meta property="og:locale" content="en_US">
   <meta property="og:site_name" content="{{ site.title }}">
   <meta property="og:title" content="{{ title }}">
   <meta property="og:url" content="{{ page.url | replace: '.html', '' }}">
@@ -15,7 +15,7 @@
   <meta name="twitter:description" content="{{ site.description }}">
   <meta name="twitter:url" content="{{ page.url | replace: '.html', '' }}">
   <meta name="twitter:card" content="summary">
-  <meta name="twitter:image" content="https://eslint.org/assets/img/favicon.512x512.png">
+  <meta name="twitter:image" content="https://eslint.org/assets/img/favicon.512x512.png"> -->
   {% if title != site.title %}
       {% if title != "ESLint" %}
           <title>{{ title }} - {{ site.title }}</title>
@@ -25,17 +25,17 @@
   {% else %}
     <title>{{ site.title }}</title>
   {% endif %}
-  <link rel="preconnect" href="https://www.google-analytics.com">
-  {% if original %} 
+  <!-- <link rel="preconnect" href="https://www.google-analytics.com"> -->
+  <!-- {% if original %}
     <link rel="canonical" href="{{ original.url }}">
-  {% else %}  
+  {% else %}
     <link rel="canonical" href="{{ page.url | replace: '.html', '' }}">
-  {% endif %}  
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600">
-  <link rel="stylesheet" href="/assets/styles/main.css">
-  <link rel='manifest' href='/manifest.json'>
-  <link rel="icon" href="/assets/img/favicon.512x512.png">
-  <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed.xml">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css">
-  <script async src="https://media.ethicalads.io/media/client/ethicalads.min.js"></script>
+  {% endif %} -->
+  <!-- <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600"> -->
+  <link rel="stylesheet" href="{{ page | relativePath }}assets/styles/main.css">
+  <!-- <link rel='manifest' href='/manifest.json'> -->
+  <!-- <link rel="icon" href="/assets/img/favicon.512x512.png"> -->
+  <!-- <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed.xml"> -->
+  <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css"> -->
+  <!-- <script async src="https://media.ethicalads.io/media/client/ethicalads.min.js"></script> -->
 </head>
