diff --git a/_pages/index.liquid b/_pages/index.liquid
index 09902aa3..2a266941 100644
--- a/_pages/index.liquid
+++ b/_pages/index.liquid
@@ -20,7 +20,7 @@ homepage: true
     </div>
 </div>
 
-<hr class="hr-near-below">
+<!--hr class="hr-near-below">
 
 <div class="sponsors row">
   <div class="col-sm-12">
@@ -58,7 +58,7 @@ homepage: true
     </div>
     <p class="sponsor-btn text-center"><a class="btn btn-default" href="https://opencollective.com/eslint">Become a Sponsor today!</a></p>
   </div>
-</div>
+</div> -->
 
 <hr class="hr-near-below">
 
