diff --git a/.eleventyignore b/.eleventyignore
index c6ececca860091e0c6e31670f4b3f6efe7db7081..fb013cf5d167cc6f256782690c343d63a4472d80 100644
--- a/.eleventyignore
+++ b/.eleventyignore
@@ -5,3 +5,25 @@ package-lock.json
 package.json
 README.md
 webpack.config.js
+
+# dash-docset-gen
+_pages/404*
+_pages/blog*
+_pages/cla*
+_pages/demo*
+_pages/feed*
+_pages/team*
+_pages/users*
+_posts
+docs/0.24.1
+docs/1.0.0
+docs/1.10.3
+docs/2.0.0
+docs/2.13.1
+docs/3.0.0
+docs/4.0.0
+docs/5.0.0
+docs/6.0.0
+docs/7.0.0
+docs/8.0.0
+docs/about
